suites/pacific/rgw/tier_1_object.yaml

```diff
@@ -124,4 +124,13 @@ tests:
         script-name: test_swift_basic_ops.py
         config-file-name: test_swift_object_expire_op.yaml
         timeout: 500
-        
+   - test:
+      name: object lock verification
+      desc: object lock test in s3
+      polarion-id: CEPH-83574055
+      module: sanity_rgw.py
+      config:
```

Comment:
I think you need to give an additional space...
```
- test:
    name: object lock verification
    desc: ...
```

Reply:
Modified it accordingly to align with existing yaml.